src/sentry/tasks/store.py

```diff
@@ -132,8 +143,22 @@ def _do_preprocess_event(cache_key, data, start_time, event_id, process_task, pr
 
     if should_process_with_symbolicator(data):
         reprocessing2.backup_unprocessed_event(project=project, data=original_data)
-        submit_symbolicate(
-            project, from_reprocessing, cache_key, event_id, start_time, original_data
+
+        is_low_priority = killswitch_matches_context(
+            "store.symbolicate-event-lpq-always",
+            {
+                "project_id": project_id,
+            },
+        )
```

Comment:
This doesn't take the `store.symbolicate-event-lpq-never` override into account; is that intentional?

Reply:
It made sense to me when I wrote these as I figured `-never` doesn't really take effect until `-selected` makes it in; If we're manually configuring these then chances are we're probably just inserting and removing project IDs into `-always` to switch between queues as needed.

I've just realized that reasoning is a little flawed since there's nothing stopping whoever it is that configures these options to put the same project in both kill switches. Thanks for bringing this up!

Reply:
yeah, i agree we might as well start checking both now